microsite/data/plugins/cost-insights.yaml

```diff
@@ -5,7 +5,7 @@ authorUrl: https://github.com/spotify
 category: Discovery
 description: Visualize, understand and optimize your team's cloud costs.
 documentation: https://github.com/backstage/backstage/tree/master/plugins/cost-insights
-iconUrl: https://www.materialui.co/materialIcons/editor/monetization_on_white_192x192.png
+iconUrl: static/img/cost-insights-plugin-icon.png
```

Comment:
This doesn't exist in the repo yet right? So probably need to add the file too, and with that I believe it should be just `img/` instead of `static/img/`.